listenbrainz/webserver/static/js/src/types.d.ts

```diff
@@ -453,6 +453,10 @@ declare type RecommendationFeedbackMap = {
   [recordingMbid: string]: RecommendationFeedBack | null;
 };
 
+/** ***********************************
```

Comment:
this needs to be below the definition of `UserTrackRecommendationMetadata`

Reply:
I sort of thought it was similar territory, but I'll move it.